app/models/page_sequence.rb

```diff
@@ -25,6 +25,17 @@ def next_slug
     slugs[current_slug_index + 1]
   end
 
+  def previous_slug
+    slug_index = current_slug_index
+    deadend_slugs = %w[complete existing-session eligible-now eligibility-confirmed eligible-later ineligible]
+
+    if slug_index.zero? || current_slug.in?(deadend_slugs)
```

Comment:
It's a personal preference but I usually use a guard clause on a conditional that ends a method.

```ruby
return nil if slug_index.zero? || current_slug.in?(deadend_slugs)

slugs[slug_index - 1]
```

Reply:
Yeah I prefer that as well, short circuit first

Reply:
Done, but it's one of the things I like about Ruby: you don't need to explicitly `return`

Reply:
Just a style, I'm not too fussed just convention and rubocop does pick these up on certain scenarios